CodeQL.yml

```diff
@@ -0,0 +1,5 @@
+# See https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/codeql/troubleshooting/bugs/generated-library-code
+path_classifiers:
+  # Tagging the sqlite folder as 'library'.
+  library:
+    - "sqlite/**"
```

Comment:
Will mark sqlite as 'library' exclude it from the scanning result?

Reply:
Yes, I believe so.

Reply:
But why exclude sqlite from the result? as we forked it, we need make sure the fork is clean on CodeQL, is this right?

Reply:
The CodeQL support team provided documentation for addressing CodeQL issues in library code: [CodeQL Troubleshooting - Bugs in Generated Library Code](https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/codeql/troubleshooting/bugs/generated-library-code). I noticed that a couple of teams in MSO are using this configuration to mark the library files.

Alternatively, we could move the `memset` function to `memset_s`, as I proposed in my previous PR: [PR #1286 - Move to memset_s](https://github.com/microsoft/cpp_client_telemetry/pull/1286).

Please let me know which solution you think would be better.